packages/discovery-provider/src/queries/search_es.py

```diff
@@ -386,23 +340,61 @@ def track_dsl(
 ):
     dsl = {
         "must": [
-            *base_match(search_str),
             {"term": {"is_unlisted": {"value": False}}},
             {"term": {"is_delete": False}},
+            {
+                "bool": {
+                    "should": [
+                        *base_match(search_str),
+                        {"wildcard": {"title": "*" + search_str + "*"}},
```

Comment:
Nice so this covers prefix and suffix?

Reply:
Additionally, it looks like case insensitivity and boost is applied for artist case for this, do we want to be consistent or is this intentional?

Reply:
oh true ill add it to the others